tutor/src/components/payments/manage.jsx

```diff
@@ -53,6 +123,12 @@ export default class ManagePayments extends React.PureComponent {
           <BackButton fallbackLink={this.backLink} />
         </header>
         {Purchases.hasApiRequestPending ? <OXFancyLoader isLoading /> : this.renderTable()}
+        <div className="refund-policy">
+          <NewTabLink to="http://openstax.force.com/support/articles/FAQ/OpenStax-Tutor-Student-Refund-Policy/?q=refund&l=en_US&c=Products%3ATutor&fs=Search&pn=1">
```

Comment:
hm, yeah also would be nice if they came from the BE, that way we could have different one's for -dev vs production